### ISSUE TITLE
Future of the project

### ISSUE THREAD

**Issue by demisx** [kriskbx/gitlab-time-tracker#109]
Can you guys please share your future plans/vision for this project? Are you planning on maintaining it 🤞, is it abandoned at this point, etc? This is a great value add to GitLab and thank you everyone who contributed to it.

**Comment by kriskbx** [comment]
To be honest: I don't have any time right now and in the foreseeable future to work on this project. I'm very sorry. If anyone is interested in taking over or at least in moderating issues and accepting PRs please drop me a line. Any help is really appreciated!

**Comment by cgdobre** [comment]
Hi. I also have limited time, but maybe with our limited times combined we can keep this going. It's been a great help to me so I'd be willing to contribute. Let me know how i can help

**Comment by kriskbx** [comment]
Awesome. Let's give it a try! For starters you could test and accept/decline the open PRs. What do you think? :)

**Comment by cgdobre** [comment]
Sure. Sounds like a good start. Do you have a test plan / testing guide?

**Comment by kriskbx** [comment]
Unfortunately not. I always pull changes and test the basic functions with an empty repository. But I can try to setup a testing guide within the next weeks, if you want. Or you can. Or we can try to work on that together. What do you think? :)

**Comment by cgdobre** [comment]
Yes, let's work together on it. Can you start by sketching the list of tasks you usually do to test? I can ask or add starting form there.

I suggest we add it straight into a CONTRIBUTING.md, as any contributor may be interested in testing his/her changes too.

**Comment by kriskbx** [comment]
Good idea! I'll try to do it today or tomorrow. Thanks a lot for your help! It's really appreciated. :)

**Comment by kriskbx** [comment]
I added a CONTRIBUTING.md with a list of the most basic functions. Let's continue the discussion on that in #112.
